src/components/ClubCard.js

```diff
@@ -131,7 +130,7 @@ class ClubCard extends Component {
                         <Photo src={`/school/${data.id}.jpg`} ready={ready} />
                     </Box>
                     <Flex p={3} justify="space-around" flexDirection="column" style={{ flex: 1 }}>
-                        <Heading.h4>{data.name}</Heading.h4>
+                        <Heading.h4 style={{ textTransform: "capitalize" }}>{data.name}</Heading.h4>
```

Comment:
try the `caps` prop :)

Reply:
I don't want it all to be uppercased :stuck_out_tongue:

Reply:
oh! yes. every time I see `textTransform` I figure it's uppercase. sorry, will quietly duck out of the room now :)